torch_xla/debug/frame_parser_util.py

```diff
@@ -0,0 +1,71 @@
+import collections
+import os
+import re
+import sys
+
+
+def parse_frame_content(line):
+  # Python Frames:
+  m = re.match(r'Python Frames:', line)
+  if m:
+    return line
+  # train_loop_fn (test/test_train_mp_imagenet.py:216)
+  m = re.match(r'.*\s\(.*:\d*\)', line)
+  if m:
+    return line
+  # [Unlowered Op _local_scalar_dense from Device TPU:0]
+  m = re.match(r'\[TAG\s(.*)\sFrom Thread\s\d*\]', line)
+  if m:
+    return f'Unlowered Op: "{m.group(1)}"\n'
+
+
+def create_report(frames):
+  mrkeys = sorted(frames.keys(), key=lambda x: frames[x], reverse=True)
+  report = []
+  report.append('=' * 80)
+  report.append(
+      'Unlowered Op usage summary (more of these ops, lower performance)')
+  report.append(
+      'Note: _local_scalar_dense typically indicates CPU context access')
+  report.append('-' * 80)
+  for key in mrkeys:
+    report.append('FRAME (count={}):'.format(frames[key]))
+    report.append(f'{key}')
+    report.append('')
+  report.append('=' * 80)
+
+  report_str = '\n'.join(report)
+  if os.environ.get('PT_XLA_DEBUG_FILE'):
+    with open(os.environ.get('PT_XLA_DEBUG_FILE'), 'w') as f:
+      f.write(report_str)
+  else:
+    print(report_str)
```

Comment:
if we're printing to stdout, can we prefix the lines w/ something like `PT/XLA Profiler: `. Would be easier to grep and differentiate from the main process stdout.